lib/pages/account_page/account_page.dart

```diff
@@ -43,29 +44,11 @@ class _AccountPage extends ConsumerState<AccountPage> with Functions {
                     ),
                   ),
                   const Padding(padding: EdgeInsets.all(8.0)),
-                  const LineChartWidget(
-                    line1Data: [
-                      FlSpot(0, 3),
-                      FlSpot(1, 1.3),
-                      FlSpot(2, -2),
-                      FlSpot(3, -4.5),
-                      FlSpot(4, -5),
-                      FlSpot(5, -2.2),
-                      FlSpot(6, -3.1),
-                      FlSpot(7, -0.2),
-                      FlSpot(8, -4),
-                      FlSpot(9, -3),
-                      FlSpot(10, -2),
-                      FlSpot(11, -4),
-                      FlSpot(12, 3),
-                      FlSpot(13, 1.3),
-                      FlSpot(14, -2),
-                      FlSpot(15, -4.5),
-                      FlSpot(16, 2.5),
-                    ],
-                    colorLine1Data: Color(0xffffffff),
-                    line2Data: <FlSpot>[],
-                    colorLine2Data: Color(0xffffffff),
+                  LineChartWidget(
```

Comment:
We should probably show some days at the bottom of the chart as a reference. See the [Figma](https://figma.com/file/6NyY9yqunpbU7HIkbNEAL3/Sossoldi-App?type=design&node-id=0-1&mode=design&t=t2tpmLbyvuLyOtRZ-0)

Reply:
I agree, it is relative to the widget